files/image_config/platform/platform-init.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=First boot platform initialization
+Before=updategraph.service
+ConditionPathExists=/host/platform/firsttime
+
+[Service]
+Type=oneshot
+ExecStartPre=/bin/echo "First boot detected. Initializing platform..."
+ExecStart=/usr/bin/platform-init.sh
+ExecStartPost=/bin/rm -f /host/platform/firsttime
```

Comment:
I believe it would be possible to disable the service after it completes. I hadn't considered the possibility, as the 'firsttime' file method is currently used, and it just seems a little strange to have a service disable itself.

Anyone else have an opinion on this?